docs/eloquent-models/model-class.txt

```diff
@@ -281,6 +284,43 @@ To learn how to change the behavior when attempting to fill a field omitted
 from the ``$fillable`` array, see `Mass Assignment Exceptions <https://laravel.com/docs/{+laravel-docs-version+}/eloquent#mass-assignment-exceptions>`__
 in the Laravel docs.
 
+.. _laravel-third-party-model:
+
+Extend Third-Party Model Classes
+--------------------------------
+
+You can use {+odm-short+} to extend a third-party model class by
+including the ``DocumentModel`` trait when defining your model class. By
+including this trait, you can make the third-party class compatible with
+MongoDB.
+
+When you apply the ``DocumentModel`` trait to a model class, you must
+declare the following properties in your class:
+
+- ``$primaryKey``: defines the field that uniquely identifies a MongoDB document.
+- ``$keyType``: defines the type of the primary key value.
+
+Extended Class Example
+~~~~~~~~~~~~~~~~~~~~~~
+
+In this example, you create a ``Planet`` model class that extends the
```

Comment:
S: i'd probably go with "This example shows..." or similar. it's more consistent with the rest of our docs, plus there's no guarantee the user is actually doing this too